src/main/scala/com/kubukoz/next/Spotify.scala

```diff
@@ -103,15 +109,15 @@ object Spotify {
 
           val currentLength = player.progress_ms.millis
 
-          methods
-            .audioAnalysis[F](track.uri)
+          (implicitly[SpotifyApi[F]]
+            .getAudioAnalysis(track.uri.id): F[AudioAnalysis])
```

Comment:
FYI @baccata this might be a big deal in Scala 3, but maybe I'm just missing something. Without the explicit upcast I can't call any combinators on a method from an `F[_]`-kinded service alias.

Reply:
Ergh. I think that'd be a bug to raise with the Scala 3 codebase after minification.

Reply:
https://github.com/lampepfl/dotty/issues/14176